packages/components/src/mobile/image/index.native.js

```diff
@@ -55,7 +57,8 @@ const ImageComponent = ( {
 	retryMessage,
 	url,
 	width: imageWidth,
-	focalPoint,
+	focalPoint: focalPointValues,
+	withFocalPoint = false,
```

Comment:
I might be missing some context on this, but I was wondering if we could make this interface smaller. Do we need to differentiate between `withFocalPoint` flag being false and `focalPoint` not being passed? In any case, if it's necessary, perhaps we could call `withFocalPoint` something else.. maybe `hasFocalPoint`, or even `isCropped` if that's what it really represents?

Reply:
Thanks for bringing it up! I'll give it a thought, I agree that we could make it smaller so it'd be ideal just to have `focalpoint` and that's it.